### DISCUSSION TITLE
ElevenClock not displaying on shared screen for Microsoft Teams call

### DISCUSSION THREAD

**Discussion by brianmarantz** [martinet101/ElevenClock#1168] [General]
When I share any of my monitor screens via Microsoft Teams, ElevenClock stops displaying.  It reverts back to the standard Win11 clock until I stop sharing my screen.   Doesn't matter if its my primary screen or a secondary screen.  It only happens when you share the entire screen.  If you share just a window of a screen, ElevenClock still displays correctly--but that is because the Win11 taskbar itself isn't part of what is being shared in that situation.  It only repros if you share an entire screen which includes the Win11 taskbar and ElevenClock.

I can't seem to find a setting to correct or allow for this.  Or is this a bug?

Win11 (10.0.22000).
ElevenClock v4.1.0
3 monitor setup with ElevenClock visible on all 3 in lower right corner of Win11 taskbar
Seconds showing

**Comment by marticliment** [comment]
Hello @GeeJay360,

this is an expected behaviour, caused due to the changes on taskbar's attributes changes when Teams goes in share mode, and it can't really be fixed

**Reply by brianmarantz** [reply]
Thanks.  Yea, this used to work until sometime last month.  Some people would actually comment because when I share my screen when driving meetings, some people would notice that my taskbar clock showed seconds.  I would tell people "ElevenClock."

**Reply by marticliment** [reply]
this change was introduced on 4.1.0. My best recommendation is that you disable updates and install version 4.0.4, since the changes introduced makes avoiding this issue impossible.
